public/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="theme-color" content="#000000">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-    <title>Entitlements Reporting</title>
+    <title>product discovery</title>
```

Comment:
Not sure I've ever seen a product title that wasn't init-capped. If this is the title shouldn't it be (throughout)?

Reply:
@bclarhk

Reply:
it's the new branding standard for non-official products